packages/next/client/url.js

```diff
@@ -0,0 +1,70 @@
+import { encode, decode } from 'querystring'
+
+const IS_BROWSER = typeof window !== 'undefined'
```

Comment:
Here's a thought: why don't we just make this module browser-only, and only enable the `url` and `querystring` aliases for the browser build?  We could alias `querystring-es3` to `querystring` for the server build.

Reply:
Okay, makes sense. I will update the changes